### PR TITLE
start movement filter

### PR DESCRIPTION
**Pull request by mrow4a**

### CODE REVIEW

**Comment by ananyajh** [comment]
.

**Comment by mrow4a** [comment]
I need one change and will say when it is ready for merge

**Comment by ananyajh** [comment]
Ok.

File: lib/movementfilter.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        # Extract from file having lines in format
+        # id, timestamp, lat, long, distance, duration
+        taxi_id = current_line[0]
+        timestamp = current_line[1]
+        current_latitude = current_line[2]
+        current_longitude = current_line[3]
+        last_distance = current_line[4]
+        last_duration = current_line[5]
+
+        # TODO: filtering here and writing to file processed lines
```
Comment (mrow4a) [inline lib/movementfilter.py]: This is where we should place our algorithm @ananyajh @Gabbe1993

**Comment by mrow4a** [comment]
@gabbe1993 @ananyajh you can approve and analyse the change

**Comment by Gabbe1993** [comment]
Need write access to review and merge

**Pull request merged**